frontend/pages/admin/TeamManagementPage/TeamDetailsWrapper/UsersPage/components/RemoveUserModal/RemoveUserModal.tsx

```diff
@@ -3,47 +3,47 @@ import React from "react";
 import Modal from "components/Modal";
 import Button from "components/buttons/Button";
 
-const baseClass = "remove-member-modal";
+const baseClass = "remove-user-modal";
 
 interface IDeleteTeamModalProps {
-  memberName: string;
+  userName: string;
   teamName: string;
-  isUpdatingMembers: boolean;
+  isUpdatingUsers: boolean;
   onSubmit: () => void;
   onCancel: () => void;
 }
 
-const RemoveMemberModal = ({
-  memberName,
+const RemoveUserModal = ({
+  userName,
   teamName,
-  isUpdatingMembers,
+  isUpdatingUsers,
   onSubmit,
   onCancel,
 }: IDeleteTeamModalProps): JSX.Element => {
   return (
     <Modal
-      title={"Remove team member"}
+      title={"Remove user"}
       onExit={onCancel}
       onEnter={onSubmit}
       className={baseClass}
     >
       <>
         <p>
           You are about to remove{" "}
-          <span className={`${baseClass}__name`}>{memberName}</span> from{" "}
+          <span className={`${baseClass}__name`}>{userName}</span> from{" "}
           <span className={`${baseClass}__team-name`}>{teamName}</span>.
         </p>
         <p>
-          If {memberName} is not a member of any other team, they will lose
-          access to Fleet.
+          If {userName} is not a member of any other team, they will lose access
```

Comment:
```suggestion
          If {userName} is not assigned to any other team, they will lose access
```

(cc @noahtalerman in case you have different wording in mind — we use "Assign teams" in the create/edit user modals so I think this is consistent language.)